common/src/main/java/com/wynntils/models/items/annotators/gui/IngredientPouchAnnotator.java

```diff
@@ -21,15 +21,14 @@
 import net.minecraft.world.item.Items;
 
 public final class IngredientPouchAnnotator implements GuiItemAnnotator {
-    private static final StyledText INGREDIENT_POUCH_NAME = StyledText.fromString("§6Ingredient Pouch");
+    private static final Pattern INGREDIENT_POUCH_PATTERN = Pattern.compile("§6[a-zA-Z0-9]+(?:'s)? Pouch");
     private static final Pattern INGREDIENT_LORE_LINE_PATTERN =
             Pattern.compile("^§f(\\d+) x §7([^§]*)(?:§[3567])? \\[§([8bde])✫(§8)?✫(§8)?✫§[3567]\\]$");
 
     @Override
     public ItemAnnotation getAnnotation(ItemStack itemStack, StyledText name) {
-        // DIAMOND_AXE when not empty, SNOW when empty
-        if (itemStack.getItem() != Items.DIAMOND_AXE && itemStack.getItem() != Items.SNOW) return null;
```

Comment:
I'm curious of the change between DIAMOND_AXE and IRON_HORSE_ARMOR. Have all pouches changed type to horse armor? If so, all pouches, not just ironman, would have been broken? Or are different items used for normal/ironman pouches, and now we've broken normal pouches?

Reply:
90% of the items are now either an IRON_HORSE_ARMOR or a POTION since 1.21.

Reply:
Ok, so this was completely broken since 2.1 then, not only for ironman? (Maybe I just got confused by the corresponding bug report.)